modules/dists/StencilDist.chpl

```diff
@@ -300,7 +312,9 @@ class LocStencilDom {
   param stridable: bool;
   var myBlock, myFluff: domain(rank, idxType, stridable);
   var NeighDom: domain(rank);
```

Comment:
While there, how about to the class comment above an example of what NeighDom looks like. An example for a simple stencil would already be great.